scripts/script.js

```diff
@@ -58,40 +61,69 @@ cocktailApp.displayDrinks = function (data) {
   this.drinkList.innerHTML = '';
   // Append click-ready cards to .drink__list
   this.drinks.forEach((drink) => {
-    const drinkName = drink.strDrink;
-    const drinkImg = drink.strDrinkThumb;
 
+    // Create list element and its HTML
     const listElement = document.createElement('li');
     listElement.id = drink.idDrink;
     listElement.classList.add('drink');
-    // Note to Jay: I believe using innerHTML is fine here, as all content is provided by the API, rather than the user. I could be wrong. Let me know what you think.
     listElement.innerHTML = `
       <div class="drink__content">
         <div class="front drink__list--result">
-          <img class="front__img drink__list--thumbnail" src="${drinkImg}/preview" alt="Photo of ${drinkName}">
-          <h3 class="front__name drink__list--name">${drinkName}</p>
+          <img class="front__img drink__list--thumbnail" loading="lazy" />
+          <h3 class="front__name drink__list--name"></h3>
         </div>
         <div class="back">
         </div>
       </div>
     `;
-    // const drinkResultList = document.createElement('li');
-    // drinkResultList.classList.add('drink__list--result');
-    
-    // const drinkResultImg = document.createElement('img');
-    // drinkResultImg.classList.add('drink__list--thumbnail');
-    
-    // const drinkResultName = document.createElement('p');
-    // drinkResultName.classList.add('drink__list--name');
-    
-    // drinkResultImg.src = `${drinkImg}`;
-    // drinkResultImg.alt = `${drinkName}`;
-    // drinkResultName.innerHTML = `${drinkName}`;
     
-    // drinkResultList.append(drinkResultImg, drinkResultName);
+    // Pass API content to list's HTML elements
+    const frontImg = listElement.querySelector('.front__img');
+    frontImg.src = `${drink.strDrinkThumb}/preview`;
+    frontImg.alt = `Photo of a ${drink.strDrink}`;
+    const frontName = listElement.querySelector('.front__name');
+    frontName.textContent = drink.strDrink;
+
+    // Add an event listener and append to the page
     listElement.addEventListener('click', cocktailApp.handleCardClick);
     cocktailApp.drinkList.append(listElement);
   });
+
+  // carousel fxn
+  buttons.forEach(button => {
+    button.addEventListener('click', () => {
+      // wanted to use another class name, but couldn't make it work
+      const offset = button.dataset.button === "next" ? 1 : -1;
+      const imgList = Array.from(cocktailApp.drinkList.children);
+      console.log(imgList[0]);
+      // imgList[0].classList.add('current__pic');
+      // imgList[0].setAttribute('data-active', 'true');
+      
+      const currentPic = cocktailApp.drinkList.querySelector('.current__pic');
+      
+      // const currentPic = cocktailApp.drinkList.querySelector('[data-active]');
+      console.log(currentPic);
+      
+      let newIndex = imgList.indexOf(currentPic) + offset;
```

Comment:
The way this function generates and uses the offset is just so clever. I love it.